### PR TITLE
Update Input additional children position based on lable position

### PR DESCRIPTION
**Pull request by moathabuhamad-cengage**
Issue: #1163 

## What I did
Update Input additional children position based on lable position

## Screenshots:
image

## Checklist 
- [x] changeset has been added
- [x] Pull request description is descriptive
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

## How to test
check Input with help link behavior based on label position and visibility

### CODE REVIEW

**Review by silvalaura — APPROVED** [review]
Looks good to me. Let's wait for @orion-cengage's approval before merging.

**Review by orion-cengage — CHANGES_REQUESTED** [review]
The button appears to be the correct size, but the icon inside is too large (24px). It needs to be smaller like it previously was (20px). I think it has taken on the size it was when it was previously INSIDE the input.

image

**Comment by orion-cengage** [comment]
Can we please expand the examples shown on the docs site to show the three variants: Top label, left label, no label.

Also, the Select component has the same ability to show a Help button, although I think it is technically done a little differently. We should at least update the title of that section of the page to also be "Help Link."

**Comment by moathabuhamad-cengage** [comment]
> The button appears to be the correct size, but the icon inside is too large (24px). It needs to be smaller like it previously was (20px). I think it has taken on the size it was when it was previously INSIDE the input.
>


its currently set to behave like the IconButton which is the `theme.iconSizes.large` 32px on large inputs and `theme.iconSizes.medium` 24px on normal ones.  

image

@orion-cengage  whats the correct behavior ? always 20px ?

**Review by orion-cengage — APPROVED** [review]

File: packages/react-magma-dom/src/components/InputBase/index.tsx
Comment (silvalaura) [inline packages/react-magma-dom/src/components/InputBase/index.tsx:421]: `getHelpLinkSVGSize` or `getHelperLinkSVGSize`

File: .changeset/feat-input-help-text.md
Comment (silvalaura) [inline .changeset/feat-input-help-text.md:5]:
```suggestion
feat(Input): Update Input additional children position based on label position
```

**Pull request merged**
